packages/rule-stylesheet-limits/README.md

```diff
@@ -0,0 +1,86 @@
+# stylesheet-limits (`@sonarwhal/rule-stylesheet-limits`)
+
+Checks if CSS exceeds known stylesheet limits
+
+## Why is this important?
+
+IE versions prior to IE 10 have limits on the number of CSS stylesheets, imports, and rules which are relatively small compared to modern browsers. Once these limits are exceeded, additional stylesheets, imports, and rules are ignored. Much larger limits exist in more recent versions, but can also cause styles to be ignored if exceeded. For more details see [Stylesheet limits in Internet Explorer][stylesheet limits].
+
+## How to use this rule?
```

Comment:
We've moved this towards the end of the file. Check the other rules to see where it is now. We should update the `new-rule` generator 🙄